.env

```diff
@@ -0,0 +1 @@
+VUE_APP_GOOGLE_MAPS_API_KEY=AIzaSyDZrAywcnCcnhBPLkbvoQTw9Lca9bZ928Q
```

Comment:
Probably best not to commit dotenv files to git (usually not recommended?) because we only have a public repo. I made an API Key using the link2link.web gmail account for temporary, we can probably use this: AIzaSyBRYt3in7LFqufRTFuG1hl7gbtgudI4XhU